src/helpers/mapSupplierToContentfulFields.spec.js

```diff
@@ -17,28 +18,16 @@ const testSupplier = {
   contactEmail: Number(99.3),
   contactSocialMedia: "99,000",
   guaranteeRating: Number("4.32"),
-  guaranteesList: "- guarantee 1\n- gurantee 2\n-guarantee 3",
-  contactInfo: "[01234 5678910](tel:012345678910)",
+  guaranteeList: "- guarantee 1\n- guarantee 2\n- guarantee 3",
+  contactInfo:
+    "[01234 5678910](tel:012345678910)\n[www.example.com](https://www.example.com)",
```

Comment:
Out of interest, how do the contact info etc fields become markdown links in the first place? Is that just their initial format from the csv upload?

Reply:
Yep that's right - we've agreed with Danny that we can change the text stored in the spreadsheet to markdown where we need to.

The plan is to make the changes together once so that he understands how to make links / lists etc and then that'll be how it gets to DEA in future quarters.